main/modes/platformer/mode_platformer.c

```diff
@@ -307,51 +308,19 @@ void updateTitleScreen(platformer_t *self)
 
     // Handle inputs
     if (
-        ((self->gameData.btnState & BTN_B) && !(self->gameData.prevBtnState & BTN_B)) 
-        ||
-        ((self->gameData.btnState & BTN_A) && !(self->gameData.prevBtnState & BTN_B))
```

Comment:
I guess this had been a typo, but it's gone now!

Reply:
It definitely was.